src/test/java/io/ebean/dbmigration/DbMigrationGenerateTest.java

```diff
@@ -0,0 +1,109 @@
+package io.ebean.dbmigration;
+
+import io.ebean.BaseTestCase;
+import io.ebean.EbeanServer;
+import io.ebean.EbeanServerFactory;
+import io.ebean.Platform;
+import io.ebean.config.ServerConfig;
+
+import org.junit.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+
+/**
+ * This is the Migrationscript generator. It generates 3 migrationscript for the models
+ * @author Roland Praml, FOCONIS AG
+ *
+ */
+public class DbMigrationGenerateTest {
+
+  private static final Logger logger = LoggerFactory.getLogger(DbMigrationGenerateTest.class);
+
+  @Test
+  public void invokeTest() throws IOException {
+    main(null);
+  }
+  
+  public static void main(String[] args) throws IOException {
```

Comment:
This test case (also runnable as normal java application) generates the reference-sqls in src/test/resources/dbmigration/migrationtest